docs/application/dotnet/guides/personal/overview.md

```diff
@@ -13,10 +13,6 @@ You can use the following personal data features in your .NET applications:
 
     You can synchronize service application data between a server and the device by creating various sync jobs.
 
--   [Authentication and Authorization](authentication.md)
-
-    You can obtain an access token by using the OAuth 2.0 authorization. The OAuth 2.0 authorization framework enables you to obtain limited access to an HTTP service, either on behalf of a resource owner by orchestrating an approval interaction between the resource owner and the HTTP service, or by allowing you to obtain access on your own behalf. You can also use the FIDO Universal Authentication Framework to authenticate users.
```

Comment:
Oops.. this link should be kept as it is going to authentication.md, which remains after merging this PR. I asked to update just the sentence, as the sentence is mentioning OAuth 2.0 and FIDO Authentication.

Please revert back the removal of this hyperlink, and update the following sentence to mention FIDO Universal Authentication Framework only.


If possible, enhancing the description for FIDO Universal Authentication Framework would be appreciate...

Reply:
Changes are done, kindly review

Reply:
The enhancing is more than my expectation. As the introducing, we don't need to use heavy sentences here.

How about below?

You can authenticate users without entering password using FIDO Universal Authentication Framework.